cmd/solana_exporter/slots.go

```diff
@@ -331,6 +334,14 @@ func (c *SlotWatcher) fetchAndEmitSingleFeeReward(
 ) error {
 	block, err := c.client.GetBlock(ctx, rpc.CommitmentConfirmed, slot)
 	if err != nil {
+		var rpcError *rpc.RPCError
+		if errors.As(err, &rpcError) {
+			// this is the error code for slot was skipped:
+			if rpcError.Code == -32007 && strings.Contains(rpcError.Message, "skipped") {
+				klog.Infof("slot %v was skipped, no fee rewards.", slot)
```

Comment:
Warnf maybe?

Reply:
I mean, this part of the exporter is not designed for tracking skipped slots so in theory this is not necessarily a bad thing. For instance, we might want to track the fee rewards of Triton 1 to see if they are doing something we aren't (or just a handful of validator benchmarks), so does it make sense for us to get a warning when they have a skipped slot?